xcodeproj/xcodeproj.bzl

```diff
@@ -1,5 +1,8 @@
 """Public rules, macros, and libraries."""
 
+# Re-export original rules rather than their wrapper macros
+# so that stardoc documents the rule attributes, not an opaque
+# **kwargs argument
```

Comment:
This should be in the `doc.bzl` file, right?

Reply:
Good catch